contrib/bentoml/README.md

```diff
@@ -1,193 +1,209 @@
+Starting with the release of Kubeflow 1.7, BentoML provides a native integration with Kubeflow through [Yatai](https://github.com/bentoml/yatai-deployment). This integration allows you to package models trained in Kubeflow notebooks or pipelines as [Bentos](https://docs.bentoml.org/en/latest/concepts/bento.html), and deploy them as microservices in a Kubernetes cluster through BentoML's cloud native components and custom resource definitions (CRDs). This documentation provides a comprehensive guide on how to use BentoML and Kubeflow together to streamline the process of deploying models at scale.
 
 ## Requirements
 
 * Kubernetes 1.20 - 1.24
 
 ## Installation
 
-    * The yaml assumes you will install in kubeflow namespace
+Run the following command to install BentoML Yatai. Note that the YAML assumes you will install in kubeflow namespace.
 
 ```bash
 kustomize build bentoml-yatai-stack/default | kubectl apply -n kubeflow --server-side -f -
 ```
 
-## Upgrating
+## Upgrading
 
 See [UPGRADE.md](UPGRADE.md)
 
-# Design Proposal
-
 ## Why BentoML
 
 ![image](https://user-images.githubusercontent.com/861225/212856116-bf873dc8-7da3-4484-9f33-e401e34a82dc.png)
 
-- BentoML is an open-source framework for developing, serving, and deploying ML services.
-    - Building
-        - Unifies ML frameworks with out-of-the-box implementation of popular frameworks
-        - Exposes gRPC and OpenAPI for serving
-        - Provides Python SDK for development
-    - Deployment
-        - Any environment, batch inference, streaming, or online serving
-        - Any cloud platform for on-prem
-        - Full observability support through Grafana
-        - Yatai - BentoML's deployment platform
-
-## User Stories
-
-Goal: From simple Python module to distributed Kubernetes deployment.
+[BentoML](https://github.com/bentoml/BentoML) is an open-source platform for building, shipping, and scaling AI applications.
 
-Consider the following common ML services involve custom pre and post-processing logic and inference of multiple models.
+- Building
+    - Unifies ML frameworks to run inference with any pre-trained models or bring your own
+    - Multi-model Inference graph support for complex AI solutions
+    - Python first framework that integrates with any ecosystem tooling
+- Shipping
+    - Any environment, batch inference, streaming, or real-time serving
+    - Any public cloud for on-prem deployment
+    - Kubenetes native deployment
+- Scaling
+    - Efficient resource utilization with autoscaling
+    - Adaptive batching for higher efficiency and throughput
+    - Distributed microservice architecture to run services on the most optimal hardware
 
-![image](https://user-images.githubusercontent.com/861225/212856456-866125c8-2bf3-42d4-b031-3c7d89c07f37.png)
+## Workflow on Kubeflow Notebook
 
-### Developing on Kubeflow Notebook
```

Comment:
Please name it Workbenches as that will be the case in 1.8

Reply:
Since this documentation is going to be used for 1.7, would you still like me to update the naming from Notebook to Workbench?

Reply:
i would prefer it. @DomFleischmann @kimwnasptd  what do you think?

Reply:
@jbottum

Reply:
I would suggest using Notebooks for now.